aries-site/src/components/content/CollapsibleSection.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Collapsible, Text } from 'grommet';
+import { FormUp, FormDown } from 'grommet-icons';
+
+export const CollapsibleSection = ({ label, onClick, ...rest }) => {
+  const [open, setOpen] = useState(false);
+  return (
+    <>
+      <Box
+        background="background-contrast"
+        direction="row"
+        gap="small"
+        onClick={() => {
+          setOpen(!open);
+          if (onClick) onClick();
+        }}
+        margin={{ bottom: 'xxsmall' }}
+        pad="medium"
+        round={!open ? 'small' : { corner: 'top', size: 'small' }}
+      >
+        {!open ? <FormDown /> : <FormUp />}
+        <Text weight="bold">{label}</Text>
+      </Box>
+      <Collapsible open={open}>
+        <Box
+          background="background-contrast"
+          height={{ max: 'medium' }}
```

Comment:
Wondering if we can just have the box size to its contents and not restrict the height because the overflow scroll behavior is a little awkward? Can always add it back in if content becomes unruly.

Soft opinion.

Reply:
Good point. I bumped up the max height to large to eliminate some of the awkwardness of things that were just a tiny bit longer than 'medium'. But I think some restriction is good otherwise code examples like Tabs get super long.